src/main/java/org/craftercms/studio/impl/v2/service/policy/validators/MimeTypePolicyValidator.java

```diff
@@ -40,15 +41,34 @@ public class MimeTypePolicyValidator implements PolicyValidator {
     public static final String CONFIG_KEY_MIME_TYPES = "mime-types";
 
     @Override
-    public void validate(HierarchicalConfiguration<?> config, Action action) throws ValidationException {
-        if (config.containsKey(CONFIG_KEY_MIME_TYPES) && !isEmpty(FilenameUtils.getExtension(action.getTarget()))) {
+    public void validate(HierarchicalConfiguration<?> permittedConfig, HierarchicalConfiguration<?> deniedConfig, Action action) throws ValidationException {
+        if (isEmpty(FilenameUtils.getExtension(action.getTarget()))) {
+            logger.debug("Skipping folder '{}'", action.getTarget());
+            return;
+        }
+
+        var isContainPermitConfig = permittedConfig.containsKey(CONFIG_KEY_MIME_TYPES);
```

Comment:
@phuongnq please change the name to `containsPermittedConfig`, the `is` prefix is usually used only for getter methods